### DISCUSSION TITLE
Github account recovery

### DISCUSSION THREAD

**Discussion by mifraheem** [community/community#142142] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### GitHub Feature Area

Actions

### Body

Dear GitHub Support,

I am writing to request assistance in recovering my GitHub account associated with my company email [redacted by moderator for PII]. Unfortunately, I have forgotten the account password, and when I try to reset it, I receive an error message stating "invalid email or not primary email."

To the best of my knowledge, this email should be the primary or at least associated with the account. However, I am currently unable to reset my password.

Here are the details of my account:

Account Email: [redacted by moderator for PII]
Company Name: LogicExer
GitHub Username: logicexer
I would appreciate your help in either confirming the primary email address associated with the account or guiding me through the process of resetting my password. If additional information or verification is needed, I am happy to provide it.

Thank you for your assistance.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mifraheem,  

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

_Please note: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
